app/src/main/res/layout/main_activity.xml

```diff
@@ -43,6 +43,7 @@
 
         <FrameLayout
             android:id="@+id/main"
+            android:background="@color/splashscreen_background"
```

Comment:
Why are you setting the background of MainActivity to splashscreen's background?